index.html

```diff
@@ -0,0 +1,370 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="mobile.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="setup">
+      <header>
+        <a href="">My Log</a>
+        <nav>
+          <img
+            src="./images/Vector.png"
+            alt="image was moved to another location"
+          />
+        </nav>
+      </header>
+      <section>
+        <div class="bg">
+          <img id="Ellipse11" src="./images/Ellipse11.png" alt="ellipse11" />
+          <img id="Ellipse3" src="images/Ellipse3.png" alt="ellipse3" />
+          <img id="Rectangle4" src="./images/Rectangle4.png" alt="rectangle4" />
+        </div>
```

Comment:
- [ Optional ] we apologize for not point this out earlier. However, this images are part of a bigger background image in the Figma design and the that `background-image` should be added as CSS background-image to the `headline` section instead. You can export the background-image from the Figma design as svg and you can find it under the `Template-1-Mobile-Main` with the name of `header-shapes Mobile`. Please have a look at this [link](https://www.w3schools.com/cssref/pr_background-image.asp) for how to add background images.